src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -17,7 +17,7 @@
 import teammates.common.util.StatusMessageColor;
 import teammates.ui.pagedata.AdminEmailComposePageData;
 
-public class AdminEmailComposeSendAction extends Action {
+public final class AdminEmailComposeSendAction extends Action {
```

Comment:
yeah your right, but shouldn't the class be final since it doesn't extend any subclass?